### PR TITLE
Made the ie-to-edge redirect page less confusing

### LINKED ISSUE #1718: IE compatibility List conditions to add a website are confusing
> The IE compatibility List only adds a website when the following actions occur.
> - Shows an IE user a message suggesting the user should use a different browser for compatibility reasons.
> - Owner requests to add the website to the IE compatibility list.

It is not clear to me if both actions need to occur or only one of them.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 6fe5c21b-2296-8b5c-208a-f41eb967c24e
* Version Independent ID: c3722063-7455-5ba9-a78d-102823dfd474
* Content: [Moving users to Microsoft Edge from Internet Explorer - Microsoft Edge Development](https://docs.microsoft.com/en-us/microsoft-edge/web-platform/ie-to-microsoft-edge-redirection)
* Content Source: [microsoft-edge/web-platform/ie-to-microsoft-edge-redirection.md](https://github.com/MicrosoftDocs/edge-developer/blob/main/microsoft-edge/web-platform/ie-to-microsoft-edge-redirection.md)
* Product: **microsoft-edge**
* GitHub Login: @MSEdgeTeam
* Microsoft Alias: **msedgedevrel**

### PR DESCRIPTION
**Pull request by captainbrosset**
This PR aims at making the "Move users to Microsoft Edge from Internet Explorer" page less confusing and fixes #1718.

The way the page is currently written makes it seem like IE displays an incompatibility message to customers and that this message is one of the required steps before being granted a redirection.
However this isn't correct. IE does not show any messages at all. Websites may sometimes do so, but it's up to them.
And website owners can decide to register their URLs with the IE compatibility list whenever they want.

Hopefully this is better.

Before: https://docs.microsoft.com/en-us/microsoft-edge/web-platform/ie-to-microsoft-edge-redirection

After: https://review.docs.microsoft.com/en-us/microsoft-edge/web-platform/ie-to-microsoft-edge-redirection?branch=pr-en-us-1725

### CODE REVIEW

File: microsoft-edge/web-platform/ie-to-microsoft-edge-redirection.md [hunk 1]
```diff
@@ -5,28 +5,15 @@ author: MSEdgeTeam
...
-
-
-<!-- ====================================================================== -->
-## Why is a website added to the Internet Explorer compatibility list?
-
-The Internet Explorer compatibility List only adds a website when the following actions occur.
-
-*   Shows an Internet Explorer user a message suggesting the user should use a different browser for compatibility reasons.
-*   Owner requests to add the website to the Internet Explorer compatibility list.
+To minimize disruptions Microsoft Edge supports a new capability that automatically redirects users.  When an Internet Explorer user goes to a website that's incompatible with Internet Explorer, Windows can automatically redirect the user to Microsoft Edge.  Only websites that are part of the [Need Microsoft Edge list](https://edge.microsoft.com/neededge/v1) are redirected.
```
Comment (mikehoffms) [inline microsoft-edge/web-platform/ie-to-microsoft-edge-redirection.md:16]:
```suggestion
To minimize disruptions Microsoft Edge supports a new capability that automatically redirects users.  When an Internet Explorer user goes to a website that's incompatible with Internet Explorer, Windows can automatically redirect the user to Microsoft Edge.  Only websites that are part of the [Need Microsoft Edge list](https://edge.microsoft.com/neededge/v1) are redirected.
```
```suggestion
To minimize disruptions, Microsoft Edge supports a new capability that automatically redirects users.  When an Internet Explorer user goes to a website that's incompatible with Internet Explorer, Windows can automatically redirect the user to Microsoft Edge.  Only websites that are part of the [Need Microsoft Edge list](https://edge.microsoft.com/neededge/v1) are redirected.
```

File: microsoft-edge/web-platform/ie-to-microsoft-edge-redirection.md [hunk 2]
```diff
@@ -5,28 +5,15 @@ author: MSEdgeTeam
...
-ms.date: 11/13/2020
+ms.date: 2/8/2022
 ---
 # Move users to Microsoft Edge from Internet Explorer
 
-Many modern websites have designs that are incompatible with Internet Explorer.  When an Internet Explorer user visits an incompatible public website, the user may get a message.  The message states that the website is incompatible with the browser.  After the message is displayed, the user is expected to manually switch to a modern browser.
+Many modern websites have designs that are incompatible with Internet Explorer.
 
-To minimize disruptions, starting with version 84, Microsoft Edge supports a new capability that automatically redirects users.  When an Internet Explorer user goes to a website that's incompatible with Internet Explorer, Windows automatically redirects the user to Microsoft Edge.  For the list of websites that are redirected, see [Need Microsoft Edge list](https://edge.microsoft.com/neededge/v1).
+When an Internet Explorer user visits an incompatible public website, the user may be instructed by the website that it is incompatible with Internet Explorer and to switch to a more modern browser.
```
Comment (mikehoffms) [inline microsoft-edge/web-platform/ie-to-microsoft-edge-redirection.md:14]:
```suggestion
When an Internet Explorer user visits an incompatible public website, the user may be instructed by the website that the website is incompatible with Internet Explorer, and that the user must switch to a more modern browser in order to use the website.
```
"that it is" and "to switch" aren't parallel enough, IMO; hard to parse.  "It" is less clear than could be.

**Review by mikehoffms — APPROVED** [review]

**Review by vchapel — APPROVED** [review]
LGTM
